### PR TITLE
Breaking change: Remove deprecated prometheus metrics from Metric Server

### PR DESCRIPTION
**Pull request by JorTurFer**
_Provide a description of what has been changed_

### Checklist

- [x] When introducing a new scaler, I agree with the [scaling governance policy](https://github.com/kedacore/governance/blob/main/SCALERS.md)
- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Tests have been added
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)* https://github.com/kedacore/charts/pull/432
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)* https://github.com/kedacore/keda-docs/pull/1117
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes https://github.com/kedacore/keda/issues/3930

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### Breaking Changes
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
+- Prometheus metrics on KEDA Metric Server are removed ([#3972](https://github.com/kedacore/keda/issues/3972))
```
Comment (tomkerkhove) [inline CHANGELOG.md:70]: Let's show we don't just break things:
```suggestion
- Prometheus metrics on KEDA Metric Server are removed (As per [deprecation notice](https://github.com/kedacore/keda/issues/3972) from December 2022)
```

Comment (JorTurFer) [reply CHANGELOG.md]: nice idea!

**Comment by JorTurFer** [comment]
@zroubalik will tackle this PR and the removal of the querying feature in the metrics server

**Pull request closed without merging**